src/storage/checkpoint_manager.cpp

```diff
@@ -99,9 +107,10 @@ void SingleFileCheckpointWriter::CreateCheckpoint() {
 	 */
 	BinarySerializer serializer(*metadata_writer);
 	serializer.Begin();
-	serializer.WriteList(100, "schemas", schemas.size(), [&](Serializer::List &list, idx_t i) {
-		auto &schema = schemas[i];
-		list.WriteObject([&](Serializer &obj) { WriteSchema(schema.get(), obj); });
+
+	serializer.WriteList(100, "catalog_entries", catalog_entries.size(), [&](Serializer::List &list, idx_t i) {
```

Comment:
@Maxxen this used to be "schemas" of only SchemaCatalogEntries, now it's "catalog_entries" of all CatalogEntry types, thoughts on storage stability with this change?